diff --git a/lib/index.d.ts b/lib/index.d.ts
index 2a890979ef53774dd9c592fe1a70f3900f89ed7a..3024d4535e4bcd3952f45c358998e5f6b07f48d1 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -9,7 +9,7 @@ declare global {
 export declare type ContextProps = {
     options?: {};
 };
-export declare const MathJaxProvider: React.FC<ContextProps>;
+export declare const MathJaxProvider: React.FC<React.PropsWithChildren<ContextProps>>;
 export declare function useTexSVG({ latex, onSuccess, onError, }?: {
     latex?: string | undefined;
     onSuccess?: ((html: HTMLElement) => void) | undefined;
@@ -25,6 +25,6 @@ export declare type Tex2SVGProps = {
     onError?: (html: HTMLElement) => void;
     onSuccess?: (html: HTMLElement) => void;
 };
-export declare const Tex2SVG: React.FC<Tex2SVGProps>;
-declare const Tex2SVGWithProvider: React.FC<Tex2SVGProps>;
+export declare const Tex2SVG: React.FC<React.PropsWithChildren<Tex2SVGProps>>;
+declare const Tex2SVGWithProvider: React.FC<React.PropsWithChildren<Tex2SVGProps>>;
 export default Tex2SVGWithProvider;
